app/helpers/application_helper/toolbar/ems_container_center.rb

```diff
@@ -24,6 +24,25 @@ class ApplicationHelper::Toolbar::EmsContainerCenter < ApplicationHelper::Toolba
           t = N_('Edit this Containers Provider'),
           t,),
         button(
+          :ems_container_disable,
+          'pficon pficon-trend-down fa-lg',
+          t = proc do
+                if @record.enabled?
+                  N_('Disable this Containers Provider')
+                else
+                  N_('Enable this Containers Provider')
+                end
+              end,
+          t,
+          :confirm => proc do
+            if @record.enabled?
+              N_("Warning: While this provider is disabled you will not collect ANY data from it. " \
+                 "This may cause gaps in inventory, metrics and events!")
```

Comment:
s/While this provider is disabled you will not collect ANY data from it./While this provider is disabled no data will be collected from it./